### DISCUSSION TITLE
I'm getting boosts in a language I don't speak

### DISCUSSION THREAD

**Discussion by pupeno** [mastodon/mastodon#22082] [Q&A]
Hello,

I have configured my Mastodon account with the languages I can understand (English, Spanish and Esperanto), and one of the people I follow speaks English and German. He tends to boosts a lot of posts in German that end in my feed. Why aren't they being filtered out? I think actual posts in German are correctly filtered out, but ton the boosts. Is this a bug or what am I missing?

Here's a thread I had about it, with some screenshots: https://m.pablofernandez.tech/@schmidt_fu@mstdn.social/109462339526530884

I'm new to Mastodon, so I'm not sure what else to try, search for or that I should share in the question here.

**Comment by kenara** [comment]
Eble vi devus lerni la germanan?
